kong/tools/utils.lua

```diff
@@ -797,16 +797,16 @@ end
 -- a-z, A-Z, 0-9 and '-' are allowed.
 -- @param name (string) the header name to verify
 -- @return the valid header name, or `nil+error`
-_M.validate_header_name = function(name)
+_M.validate_http_token = function(name, token_type)
```

Comment:
[Cookie name spec](https://tools.ietf.org/html/rfc2616#section-2.2) technically allows a handful of other characters, but the same is true of headers. Seems fair to generalize these rules for both. Unless there are any objections?

Reply:
for the purpose it serves, I don't see any issues with that

Reply:
Understood, and I was on the fence about this, too. Do you prefer me to just duplicate the logic for the cookie name validation? The only difference would be the error message.

Reply:
yes, let's do that for now. Let's get it compete functionality/test wise first and then look at some final things like style and stuff like this.